### DISCUSSION TITLE
[建议] 使用Win+Tab切换窗口时，可以通过上下左右方向箭控制切换

### DISCUSSION THREAD

**Discussion by MicroOps-cn** [linuxdeepin/developer-center#2917] [Features Request & Ideas | 特性请求 & 头脑风暴]
现在使用Win+Tab切换窗口只能使用tab键或鼠标来控制切换，但有时候要切换的目标窗口刚好在左边，我要么就得用鼠标来选择，要么就得一直按Tab键，感觉不是很友好，希望也可以通过上下左右方向箭控制切换

**Comment by zkitefly** [comment]
win+shift+tab

**Comment by MicroOps-cn** [comment]
> win+shift+tab

感觉还是不方便。一般来说win+tab使用大拇指和无名指配合来按，但是如果我发现需要向左切换，我得再多按一个shift, 这个时候手会很不灵活。
